parser.cpp

```diff
@@ -13,7 +13,6 @@
 #include "xos.h"
 
 namespace xos {
-
```

Comment:
Keep the blank space here.

```
Separate each non-empty group with one blank line.
```
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes